ios/RNSentry.mm

```diff
@@ -91,6 +98,10 @@ + (BOOL)requiresMainQueueSetup {
 
     [SentrySDK startWithOptions:sentryOptions];
 
+#if SENTRY_TARGET_REPLAY_SUPPORTED
+    [RNSentrySessionReplay postInit];
+#endif
+
```

Comment:
This need to move after `SentryHybridSdkDidBecomeActive` otherwise the RN convertor is not used.